src/app/flights/flights.component.html

```diff
@@ -60,7 +60,7 @@ <h3 *ngIf="!this.isLoggedIn">
     <!-- price Column -->
     <ng-container matColumnDef="price">
       <th mat-header-cell *matHeaderCellDef mat-sort-header>Price</th>
-      <td mat-cell *matCellDef="let row">{{row.price}}</td>
+      <td mat-cell *matCellDef="let row">${{row.price}}</td>
```

Comment:
this reminds me, I'll change the prices so that it always show to two decimal places